app/views/categories/index.html.erb

```diff
@@ -1,14 +1,35 @@
-<p style="color: green"><%= notice %></p>
-
-<h1>Categories</h1>
-
-<div id="categories">
-  <% @categories.each do |category| %>
-    <%= render category %>
-    <p>
-      <%= link_to "Show this category", category %>
-    </p>
-  <% end %>
+<nav class="bg-dark-blue navbar navbar-expand-lg navbar-dark">
+  <div class="container-fluid">
+    <h2 class="navbar-brand text-center mx-auto fs-2 fw-bolder
+">Categories</h2>
```

Comment:
Same thing with the categories index page. There's about 10px margin around each category card. Kindly reduce your left and right margins on mobile view so that the categories card is more visible.

![image](https://user-images.githubusercontent.com/38088818/215268102-2c0f223f-fbca-4391-870d-ddf03da0a348.png)
 
Your design:

![image](https://user-images.githubusercontent.com/38088818/215268156-9595d433-f33e-425f-82c2-f64e6d25706c.png)

- Also the correct color for the button is `#60B524`